modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -45,16 +45,12 @@ endif::[]
 |Cloned Disk
 |Provision source is a cloned disk.
 
-|Import
-|Import virtual machine from a supported provider.
-
 |Operating System
-|
-|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine. If you select `Import` as the *Provider Source*, the operating system is filled in automatically, based on the operating system of the VMware virtual machine being imported.
+|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine.
```

Comment:
Should we clarify that this list is meant to optimize for the OS that is ultimately sourced from the previous `Source` dropdown?

Reply:
Proposing this sentence (see the third sentence): 
A list of operating systems available in the cluster. This is the primary operating system for the virtual machine. The list is optimized for the operating system that was specified for the previous Source dropdown menu option.

Reply:
👍